Source/Scene/Camera.js

```diff
@@ -1499,27 +1499,43 @@ define([
 
     /**
      * Translates the camera's position by <code>amount</code> along the camera's view vector.
+     * If in 2D mode, it uses zoom2D; otherwise, uses move.
```

Comment:
These comments are what's going to appear in the public documentation, and the end user is not going to know what `zoom2D` or `move` does. Maybe something more along the lines of "When in 2D mode, this will zoom out the camera instead of translating the camera's position."